packages/legend-query-builder/src/stores/QueryBuilderResultState.ts

```diff
@@ -187,6 +187,15 @@ export class QueryBuilderResultState {
     return this.executionResult.result.rows[rowIndex]?.values ?? [''];
   };
 
+  findIsCoordinatesSelected = (
```

Comment:
```suggestion
  isCoordinatesSelected = (
```

also, this kind of function can easily be put in the renderer

Reply:
let's move all these 4 `find...` functions into component